[REV-1611] Remove unused old brand image assets

jhan217 opened the pull request:
## What are we doing?
Removing old logo images and favicons

## Why are we doing this?
Clean up unneeded old brand logos and images now that we have new ones. Doing this as a separate PR in case of client-side caching issues / missed references to these images.

## Blockers
https://github.com/edx/frontend-app-ecommerce/pull/121

julianajlk reviewed (approved):
Is this part of the rebranding work you did before?

jhan217 commented:
Yup, just cleaning up stuff we don't need anymore

dianekaplan reviewed (commented):
I'm curious what the right way is to verify a  this sort of file-removing change. (I'd imagine something like searching all edx repos for these 4 filenames?) What did you cover?

jhan217 commented:
I removed the files, see what broke on the order history page, removed that code that referenced the old file, and made sure that the page works locally.

dianekaplan commented:
> I removed the files, see what broke on the order history page, removed that code that referenced the old file, and made sure that the page works locally.

Ah! I'd recommend searching this repo for the 4 filenames to make sure there are no other spots referencing them. Doing this quickly now, I see logo.svg and favicon.ico and referenced in .env.test and .env.development, so those should be updated to use the new images
